models/Applications.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const ApplicationsSchema = new Schema(
+  {
+    id: { type: mongoose.Schema.Types.ObjectId },
+    positions: { type: String, required: true, ref: 'Id Positions' },
+    postulants: { type: String, required: true, ref: 'Id Postulants' },
+    interview: { type: String },
```

Comment:
If `interview` is an Id, it should have a `ref` attribute pointing to the entity's collection model.

Reply:
☝️

Reply:
done!